Fabric.Identity.API/Persistence/CouchDb/Stores/CouchDbUserStore.cs

```diff
@@ -35,12 +35,12 @@ public CouchDbUserStore(IDocumentDbService documentDbService, ILogger logger)
             return user?.FirstOrDefault();
         }
 
-        public Task<IEnumerable<User>> GetUsersBySubjectId(IEnumerable<string> subjectIds)
+        public Task<IEnumerable<User>> GetUsersBySubjectIdAsync(IEnumerable<string> subjectIds)
```

Comment:
Any reason this is not marked async, and we are not awaiting in the method? Is the _documentDbService.GetDocumentsById function not async?

Reply:
i just updated the method name and did not check this file to ensure it was marked async.